website/docs/engine-flink/getting-started.md

```diff
@@ -34,8 +34,9 @@ For a quick introduction to running Flink, refer to the [Quick Start](quickstart
 
 
 ## Feature Support
-Fluss only supports Apache Flink's Table API.
+Fluss supports Apache Flink's Table API and Flnk's DataStream API.
```

Comment:
Hi, there is a typo:Flnk.

Reply:
@yx91490 You can submit a hot fix pr for it.